<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Webcam</string>
   </property>
  </widget>
  <widget class="QLabel" name="camera_display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="threshhold_display">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Webcam</string>
   </property>
  </widget>
  <widget class="QLabel" name="color2_value">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Recognize Gesture:</string>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="camera_here_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera Here:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Threshold_here_text">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold Here:</string>
   </property>
  </widget>
  <widget class="QLabel" name="toggle_text">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Here:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>American Sign Language Recognition</string>
   </property>
  </widget>
  <widget class="QLabel" name="toggle_text_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Final Word Here:</string>
   </property>
  </widget>
  <widget class="QLabel" name="word">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="voice_on_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Turn voice on</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voice_off_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Turn voice off</string>
   </property>
  </widget>
  <zorder>start_button</zorder>
  <zorder>stop_button</zorder>
  <zorder>threshhold_display</zorder>
  <zorder>camera_display</zorder>
  <zorder>color2_value</zorder>
  <zorder>text</zorder>
  <zorder>camera_here_text</zorder>
  <zorder>Threshold_here_text</zorder>
  <zorder>toggle_text</zorder>
  <zorder>label</zorder>
  <zorder>toggle_text_2</zorder>
  <zorder>word</zorder>
  <zorder>voice_on_button</zorder>
  <zorder>voice_off_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
